server/src/org/apache/cloudstack/affinity/AffinityGroupServiceImpl.java

```diff
@@ -323,12 +330,12 @@ public void doInTransactionWithoutResult(TransactionStatus status) {
         List<String> types = new ArrayList<String>();
 
         for (AffinityGroupProcessor processor : _affinityProcessors) {
-                if (processor.isAdminControlledGroup()) {
-                    continue; // we dont list the type if this group can be
-                              // created only as an admin/system operation.
-                }
-                types.add(processor.getType());
+            if (processor.isAdminControlledGroup()) {
```

Comment:
At first I tried to submit without those changes in lines which I did not touch; However, it failed to pass in check style. When I let my environment make the code formatting, then all checks passed. I can review those changes and see if I can minimize them. But some will be necessary.